cli/src/cli_util.rs

```diff
@@ -2135,6 +2135,9 @@ pub struct GlobalArgs {
     /// Enable debug logging
     #[arg(long, global = true)]
     pub debug: bool,
+    /// Reduce the amount of output logged
+    #[arg(long, global = true)]
+    pub quiet: bool,
```

Comment:
I expect the global `--quiet` will suppress most status messages ("Nothing changed.", "Rebased", "Working copy now at:", etc.) Is that the plan?

Reply:
I'm not too sure, perhaps this should be discussed. I just added it since `jj resolve` also had the `--quiet` flag for muting the list of conflicts, which I think might be handy if there are a lot of conflicted paths.

Reply:
I will rebase this after #3401 is merged.

Reply:
Rebased!